app/models/host.rb

```diff
@@ -370,109 +313,92 @@ def reset
   end
 
   def start
-    if validate_start[:available] && power_state == 'standby' && respond_to?(:vim_power_up_from_standby)
+    if power_state == "standby" && respond_to?(:vim_power_up_from_standby)
```

Comment:
NOTE all of these that check `respond_to?(:vim_something)` I feel like should move to Vmware::InfraManager::HostEsx where they are defined.  They can then `super` if for example in this case `power_state != 'standby'`